R/fit_seir.R

```diff
@@ -246,10 +247,17 @@ fit_seir <- function(daily_cases,
   stopifnot(nrow(samp_frac_fixed) == length(days))
   stopifnot(ncol(samp_frac_fixed) == ncol(daily_cases))
 
-  beta_sd <- f_prior[2]
-  beta_mean <- f_prior[1]
-  beta_shape1 <- get_beta_params(beta_mean, beta_sd)$alpha
-  beta_shape2 <- get_beta_params(beta_mean, beta_sd)$beta
+  # f_prior
+  f_seg_prior <- f_prior
+  S = length(unique(f_seg)) - 1 # - 1 because of 0 for fixed f0 before soc. dist.
```

Comment:
If you can change these to `<-` or just run `styler::style_file()`, that will make me very happy. :)